src/main/java/com/nha/abdm/wrapper/hrp/properties/ApplicationConfig.java

```diff
@@ -0,0 +1,33 @@
+package com.nha.abdm.wrapper.hrp.properties;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.stereotype.Component;
+
+@Component
+@PropertySource({"classpath:application.properties"})
+public class ApplicationConfig {
```

Comment:
The paths for different APIs can be just put in `.java` file as constants and be referred from there instead of config file. The config file can contain other details `Gateway dns end point`.

Reply:
Created a class GatewayApi for mapping the ABDM Api's.
![image](https://github.com/NHA-ABDM/ABDM-wrapper/assets/85893494/03fcee23-48d1-400f-b309-067f01b58df9)